### ISSUE TITLE
Kali 2016.2

### ISSUE THREAD

**Issue by 3n1gma30** [EnableSecurity/wafw00f#33]
d0c@Universe:~$ sudo apt-get install wafw00f
It goes through the normal install process. 
I run it and I get this.
d0c@Universe:~$ sudo wafw00f -l
Traceback (most recent call last):
  File "/usr/local/bin/wafw00f", line 4, in <module>
    from wafw00f import main
  File "/usr/local/lib/python3.5/dist-packages/wafw00f/main.py", line 53, in <module>
    from wafw00f.lib.evillib import oururlparse, scrambledheader, waftoolsengine
  File "/usr/local/lib/python3.5/dist-packages/wafw00f/lib/evillib.py", line 14, in <module>
    from bs4 import BeautifulSoup
  File "/usr/local/lib/python3.5/dist-packages/bs4/__init__.py", line 30, in <module>
    from .builder import builder_registry, ParserRejectedMarkup
  File "/usr/local/lib/python3.5/dist-packages/bs4/builder/__init__.py", line 314, in <module>
    from . import _html5lib
  File "/usr/local/lib/python3.5/dist-packages/bs4/builder/_html5lib.py", line 70, in <module>
    class TreeBuilderForHtml5lib(html5lib.treebuilders._base.TreeBuilder):
AttributeError: module 'html5lib.treebuilders' has no attribute '_base'
 Help?!?! Please...

**Comment by sandrogauci** [comment]
will test and let you know. Note that there is no need to use sudo with wafw00f.

**Comment by sandrogauci** [comment]
I just got Kali 2016.2 light version  for Virtualbox running and did the following:

```bash
$ wafw00f
-bash: wafw00f: command not found
$ apt update && apt install -y wafw00f
...
$ wafw00f -l 

                                 ^     ^
        _   __  _   ____ _   __  _    _   ____
       ///7/ /.' \ / __////7/ /,' \ ,' \ / __/
      | V V // o // _/ | V V // 0 // 0 // _/
      |_n_,'/_n_//_/   |_n_,' \_,' \_,'/_/
                                <
                                 ...'

    WAFW00F - Web Application Firewall Detection Tool

    By Sandro Gauci && Wendel G. Henrique

Can test for these WAFs:

Anquanbao
Juniper WebApp Secure
IBM Web Application Security
Cisco ACE XML Gateway
F5 BIG-IP APM
360WangZhanBao
... etc etc
```
I'm going to mark this as an invalid bug report. Please reopen if you cannot fix this on your own. Do note that we do not package wafw00f for Kali Linux.

**Comment by no1deer** [comment]
I am having this exact issue.

➜  ~ uname -a
Linux PTL01 4.9.0-kali4-amd64 #1 SMP Debian 4.9.25-1kali1 (2017-05-04) x86_64 GNU/Linux

➜  ~ dpkg -l wafw00f
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                         Version             Architecture        Description
+++-============================-===================-===================-==============================================================
ii  wafw00f                      0.9.3-0kali2        all                 tool to identify and fingerprint Web Application Firewall prod

➜  ~ wafw00f 
Traceback (most recent call last):
  File "/usr/local/bin/wafw00f", line 4, in <module>
    __import__('pkg_resources').run_script('wafw00f==0.9.4', 'wafw00f')
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 739, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1494, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/wafw00f-0.9.4-py2.7.egg/EGG-INFO/scripts/wafw00f", line 4, in <module>
    from wafw00f import main
  File "/usr/local/lib/python2.7/dist-packages/wafw00f-0.9.4-py2.7.egg/wafw00f/main.py", line 53, in <module>
    from wafw00f.lib.evillib import oururlparse, scrambledheader, waftoolsengine
  File "/usr/local/lib/python2.7/dist-packages/wafw00f-0.9.4-py2.7.egg/wafw00f/lib/evillib.py", line 14, in <module>
    from bs4 import BeautifulSoup
  File "build/bdist.linux-x86_64/egg/bs4/__init__.py", line 30, in <module>
  File "build/bdist.linux-x86_64/egg/bs4/builder/__init__.py", line 314, in <module>
  File "build/bdist.linux-x86_64/egg/bs4/builder/_html5lib.py", line 70, in <module>
AttributeError: 'module' object has no attribute '_base'

**Comment by haughsofwar** [comment]
I am also having the same issue
Linux kali 4.9.0-kali4-amd64 #1 SMP Debian 4.9.30-2kali1 (2017-06-22) x86_64 GNU/Linux
wafw00f  0.9.3-0kali2

**Comment by sandrogauci** [comment]
fixed #38
